File: 3830-find-closest-person/3830-find-closest-person.py
class Solution(object):
    def findClosest(self, x, y, z):
        """
        :type x: int
        :type y: int
        :type z: int
        :rtype: int
        """
        d1, d2 = abs(x - z), abs(y - z)
        return 1 if d1 < d2 else (0 if d1 == d2 else 2)

File: 3830-find-closest-person/3830-find-closest-person.cs
public class Solution {
    public int FindClosest(int x, int y, int z) {
        return (Math.Abs(x - z) >= Math.Abs(y - z)) ? ((Math.Abs(x - z) == Math.Abs(y - z)) ? 0 : 2) : 1;
    }
}